src/NServiceBus.Core/Outbox/OutboxSettings.cs

```diff
@@ -1,28 +1,27 @@
 ﻿namespace NServiceBus.Outbox
 {
     using System;
+    using NServiceBus.Configuration.AdvanceExtensibility;
     using Settings;
 
     /// <summary>
     /// Custom settings related to the outbox feature
     /// </summary>
-    public class OutboxSettings
+    public class OutboxSettings : ExposeSettings
```

Comment:
As @johnsimons showed this class sort of hides the extension point. You have to import the Advanced namespace to get `GetSettings`.